SPECS/netavark/netavark.spec

```
@@ -0,0 +1,256 @@
+# Building from fedora dependencies not possible
+# Latest upstream rtnetlink frequently required
+# sha2, zbus, zvariant are currently out of date
+
+%global debug_package %{nil}
+
+%global built_tag v1.0.3
+%global built_tag_strip %(b=%{built_tag}; echo ${b:1})
+%global gen_version %(b=%{built_tag_strip}; echo ${b/-/"~"})
+
+Name:          netavark
+Version:       1.0.3
+Release:       4%{?dist}
+Summary:       OCI network stack
+License:       ASL 2.0 and BSD and MIT
+Vendor:        Microsoft Corporation
+Distribution:  Mariner
+URL:           https://github.com/containers/%{name}
+# Source0:     %{url}/archive/%{built_tag}/%{version}.tar.gz
+Source0:       netavark-1.0.3.tar.gz
+# Source1:     %{url}/releases/download/%{built_tag}/%{name}-%{built_tag}-vendor.tar.gz
+Source1:       netavark-v1.0.3-vendor.tar.gz
```

Please don't use hard-coded versions for the tarballs.

```suggestion
Source0:       %{url}/archive/%{built_tag}/%{version}.tar.gz#/%{name}-%{version}.tar.gz
Source1:       %{url}/releases/download/%{built_tag}/%{name}-%{built_tag}-vendor.tar.gz
```

---

updated.